typings/lodash/lodash.d.ts

```diff
@@ -2641,6 +2641,15 @@ declare module _ {
             whereValue: W): LoDashArrayWrapper<T>;
     }
 
+    interface LoDashObjectWrapper<T> {
+        /**
+        * @see _.filter
+        **/
+        filter<T extends {}>(
+            callback: ObjectIterator<T, boolean>,
+            thisArg?: any): LoDashObjectWrapper<T>;
+    }
+
```

Comment:
Has to extend the `LoDashObjectWrapper` in order to chain `filter` function, not sure why the lodash typing file doesn't expose it. I've created a PR for this change(borisyankov/DefinitelyTyped#3631)